content/browser/webui/content_web_ui_controller_factory.cc

```diff
@@ -14,6 +14,9 @@
 #include "content/public/browser/web_contents.h"
 #include "content/public/browser/web_ui.h"
 #include "content/public/common/url_constants.h"
+#if defined(USE_AURA) && defined(OS_LINUX)
+#include "xwalk/runtime/browser/ui/webui/file_picker/file_picker_ui.h"
```

Comment:
@pozdnyakov 
There are other case that includes files from xwalk dirs:
```
joone@joone-foss:~/ssd/crosswalk/src/content$ grep -rn xwalk *
browser/web_contents/web_contents_impl.cc:120:#include "xwalk/tizen/browser/media/media_webcontents_observer.h"
browser/webui/content_web_ui_controller_factory.cc:18:#include "xwalk/runtime/browser/ui/webui/file_picker/file_picker_ui.h"
common/content_message_generator.h:76:#include "xwalk/tizen/common/media/media_player_messages.h"
content_browser.gypi:1905:        '<(DEPTH)/xwalk/tizen/browser/media/browser_mediaplayer_manager.cc',
content_browser.gypi:1906:        '<(DEPTH)/xwalk/tizen/browser/media/browser_mediaplayer_manager.h',
content_browser.gypi:1907:        '<(DEPTH)/xwalk/tizen/browser/media/media_webcontents_observer.cc',
content_browser.gypi:1908:        '<(DEPTH)/xwalk/tizen/browser/media/media_webcontents_observer.h',
content_browser.gypi:1909:        '<(DEPTH)/xwalk/tizen/browser/media/murphy_mainloop.cc',
content_browser.gypi:1910:        '<(DEPTH)/xwalk/tizen/browser/media/murphy_mainloop.h',
content_browser.gypi:1911:        '<(DEPTH)/xwalk/tizen/browser/media/murphy_resource.cc',
content_browser.gypi:1912:        '<(DEPTH)/xwalk/tizen/browser/media/murphy_resource.h',
content_browser.gypi:1913:        '<(DEPTH)/xwalk/tizen/browser/media/murphy_resource_manager.cc',
content_browser.gypi:1914:        '<(DEPTH)/xwalk/tizen/browser/media/murphy_resource_manager.h',
content_common.gypi:994:        '<(DEPTH)/xwalk/tizen/common/media/media_player_messages.h',
content_renderer.gypi:855:        '<(DEPTH)/xwalk/tizen/renderer/media/mediaplayer_impl.cc',
content_renderer.gypi:856:        '<(DEPTH)/xwalk/tizen/renderer/media/mediaplayer_impl.h',
content_renderer.gypi:857:        '<(DEPTH)/xwalk/tizen/renderer/media/renderer_mediaplayer_manager.cc',
content_renderer.gypi:858:        '<(DEPTH)/xwalk/tizen/renderer/media/renderer_mediaplayer_manager.h',
renderer/render_frame_impl.cc:175:#include "xwalk/tizen/renderer/media/renderer_mediaplayer_manager.h"
renderer/render_frame_impl.cc:176:#include "xwalk/tizen/renderer/media/mediaplayer_impl.h"
```
If it is not allowed, I have to add file-picker to under content or add a webui factory to xwalk